content/en/tracing/trace_collection/compatibility/php.md

```diff
@@ -37,48 +37,51 @@ PHP APM supports the following PHP versions (both ZTS and NTS):
 
 <div class="alert alert-info">
 <strong>Note:</strong>
-PHP 5.x is fully supported until version 0.75.0. It is now in maintenance mode and supported with security and important bug fixes until December 31, 2023.
+PHP 5.x is not supported starting version 1.0.0.
```

Comment:
And add that 0.99 (with release link here) is the last version which included it.